Mathlib/Tactic/Widget/Congrm.lean

```diff
@@ -51,4 +51,5 @@ open scoped Json in
 subexpressions in the goal.-/
 elab stx:"congrm?" : tactic => do
   let some replaceRange := (← getFileMap).rangeOfStx? stx | return
-  savePanelWidgetInfo stx ``CongrmSelectionPanel $ pure $ json% { replaceRange: $(replaceRange) }
```

Comment:
@Vtec234: I assume there was a good reason for moving the `stx` argument to be last, but it's a shame it results in more parenthesization

Reply:
@eric-wieser there was very little reason: in the core Lean version of this function it was always last, whereas in ProofWidgets4 it was first. This change is (temporarily) using the core version again instead of the PW4 version.

Reply:
Can you add to the PR description explaining why this is temporary?